README.md

```diff
@@ -19,11 +19,10 @@ on dune-elastodynamics, dune-foamgrid (version >= 2.7) providing the necessary t
 The dune-elastodynamics module can be found here:
 https://github.com/maxfirmbach/dune-elastodynamics
 
-The executable for the perpendicular-flap tutorial case can be
+The executable `dune-perpendicular-flap` for the perpendicular-flap tutorial case can be
 found in:
-`dune-precice/build-cmake/src/`
-after the module was build and needs to be copied to the solid-dune folder inside the respective
-tutorial case.
+`dune-precice-howto/build-cmake/examples/`
+after the module was built and needs to be copied to the `solid-dune` folder inside the respective tutorial case.
```

Comment:
This sentence made sense as a standalone bullet point, but does not have a reading flow. What are we trying to say here?